include/ppx/grfx/vk/vk_device.h

```diff
@@ -111,6 +111,11 @@ class Device
 
 extern PFN_vkCmdPushDescriptorSetKHR CmdPushDescriptorSetKHR;
 
+#if defined(VK_KHR_dynamic_rendering)
```

Comment:
These aren't defined with Vulkan headers coming from Android NDK 23.1.7779620 which is currently default. 
I was trying to upgrade to 25.xxx, since that's what we mention in the project README anyway, but there were some problems with that, and I wasn't sure if we want to maintain compatibility with 23.xx and Vulkan headers that don't have this. If we are fine with upgrading to 25.xxx, I can do that in a follow up PR since I need to figure out some issues with the android build related to upgrading, and then remove those `#if`s

Reply:
Oh, right. I had forgotten that Android doesn't have a stable Vulkan SDK release. My bad.